app/utils/discogs.schemas.ts

```diff
@@ -1,8 +1,10 @@
 import z from "zod";
 
 const AlbumSchema =  z.object({
+  cover_image:z.string(),
+  title: z.string().optional(),
```

Comment:
title should always be defined yeah?

```suggestion
  title: z.string(),
```

Reply:
Tested and there don't seem to be cases where there is no title